ballerina/records.bal

```diff
@@ -41,845 +41,752 @@ public type AwsCredentials record {
+    string projectionExpression?;
+    # Determines the level of detail about provisioned throughput consumption that is returned in
+    # the response. Valid Values: INDEXES | TOTAL | NONE
+    ReturnConsumedCapacity returnConsumedCapacity?;
+    # This is a legacy parameter. Use FilterExpression instead
+    map<Condition> scanFilter?;
+    # For a parallel Scan request, Segment identiﬁes an individual segment to be scanned by an application
+    # worker
+    int segment?;
+    # The attributes to be returned in the result. Valid Values: ALL_ATTRIBUTES | ALL_PROJECTED_ATTRIBUTES |
+    # SPECIFIC_ATTRIBUTES |COUNT
+    Select 'select?;
+    # For a parallel Scan request, TotalSegments represents the total number of segments into which the
+    # Scan operation will be divided
+    int totalSegments?;
+|};
 
 # Represents the response of Query operation.
-#
-# + ConsumedCapacity - The capacity units consumed by the Query operation. 
-# + Item - An Item that match the query criteria. It consists of an attribute name and the value for that attribute.
-public type QueryResponse record {
-    ConsumedCapacity? ConsumedCapacity?;
-    map<AttributeValue>? Item?;
+public type QueryOutput record {
+    # The capacity units consumed by the Query operation
+    ConsumedCapacity? consumedCapacity?;
+    # An Item that match the query criteria. It consists of an attribute name and the value for that attribute
+    map<AttributeValue>? item?;
 };
 
 # Reperesents the response of Scan operation.
-#
-# + ConsumedCapacity - The capacity units consumed by the Scan operation.
-# + Item - An Item that match the scan criteria. It consists of an attribute name and the value for that attribute.
-public type ScanResponse record {
-    ConsumedCapacity? ConsumedCapacity?;
-    map<AttributeValue>? Item?;
+public type ScanOutput record {
+    # The capacity units consumed by the Scan operation
+    ConsumedCapacity? consumedCapacity?;
+    # An Item that match the scan criteria. It consists of an attribute name and the value for that attribute
+    map<AttributeValue>? item?;
 };
 
 # Represents the response of Query or Scan operation.
-#
-# + ConsumedCapacity - The capacity units consumed by the Scan operation.
-# + Count - The number of items in the response.
-# + Items - An array of item attributes that match the scan criteria. Each element in this array consists of an
-#           attribute name and the value for that attribute.
-# + LastEvaluatedKey - The primary key of the item where the operation stopped, inclusive of the previous result set.
-#                      Use this value to start a new operation, excluding this value in the new request.
-# + ScannedCount - The number of items evaluated, before any ScanFilter or QueryFilter is applied. A high ScannedCount 
-#                  value with few, or no, Count results indicates an ineﬃcient Scan or Query operation.
 public type QueryOrScanResponse record {
```

Comment:
For some records, we ended up with `Input`/`Output`, and for some records we used `Request`/`Response`. Shall we make it consistent?